packages/frontity/templates/README.md

```diff
@@ -0,0 +1,28 @@
+## $name$
+
+This project was bootstrapped with [Frontity](https://frontity.org/).
+## Getting started
+
+```bash
+$ npm install
+$ npm run dev
+
+SERVER STARTED -- Listening @ http://localhost:3000
+  - mode: development
+  - target: module
+
+```
+
+## Build
+
+```bash
+$ npm run build
+```
+
+
+## Deployment (Vercel (now.sh))
+
+```bash
+$ npx now login
+$ npx now --prod
```

Comment:
@luisherranz @DAreRodz wouldn't this `npx now --prod` require a `now.json` as the one explained [here](https://docs.frontity.org/deployment/deploy-on-now#create-a-now-json) in order for this to work properly?
Maybe we can also add this file in this `npx frontity create` command?

Reply:
You are right, @juanmaguitar. Uhm, we can either add automatically that file of link to the documentation.

I guess adding the file is better, isn't it?

Reply:
Yes, as it is our recommended way of deploying a Frontity project (and as we have a specific package for this `@frontity/now`), I think it makes sense to directly provide this file by default in our installations

Reply:
Ok, wait, this is my fault. I didn't mean to review the README file already created, I meant to think **from scratch** what README we should provide here for new users.

For instance, I don't think adding information about a particular hosting would be nice as we are going to have many guides for many different hostings. A link to the relevant docs would be better.

Reply:
Oks, I'll suggest a better version of the `README.md`

Reply:
Thanks Juanma!

Reply:
@luisherranz @DAreRodz More detailed version of the `README.md` added

Reply:
Looks good to me. Thanks @juanmaguitar.